File: loopr-ios/Models/Market/MarketStatus.swift
//
//  Depth.swift
//  loopr-ios
//
//  Created by kenshin on 2018/3/6.
//  Copyright © 2018年 Loopring. All rights reserved.
//

import Foundation

enum MarketStatus: CustomStringConvertible {

    case active = "ACTIVE"
    case readonly = "READONLY"
    case terminated = "TERMINATED"
    case unknown = "UNKNOWN"

    var description: String {
        switch self {
        case .active:
            return "active"
        case .readonly:
            return "readonly"
        case .terminated:
            return "terminated"
        case .unknown:
            return "unknown"
        }
    }
}

File: android/walletsdk/src/main/java/leaf/prod/walletsdk/model/market/MarketStatus.java
package leaf.prod.walletsdk.model.market;

import com.google.gson.annotations.SerializedName;

/**
 * Created with IntelliJ IDEA.
 * User: kenshin wangchen@loopring.org
 * Time: 2019-03-21 2:12 PM
 * Cooperation: loopring.org 路印协议基金会
 */
public enum MarketStatus {

    @SerializedName(value = "ACTIVE")
    ACTIVE,

    @SerializedName(value = "READONLY")
    READONLY,

    @SerializedName(value = "TERMINATED")
    TERMINATED,

    UNKNOWN
}
